policy/modules/system/systemd.if

```diff
@@ -1126,3 +1126,48 @@ interface(`systemd_write_all_user_keys',`
 
 	allow $1 systemd_user_session_type:key write;
 ')
+
+########################################
+## <summary>
+##  Execute systemd-sysusers in the
+##  systemd sysusers domain.
+## </summary>
+## <param name="domain">
+##	<summary>
+##	Domain allowed access.
+##	</summary>
+## </param>
+#
+interface(`systemd_domtrans_systemd_sysusers', `
```

Comment:
I think we should drop the extra "systemd" in the interface names here and below.